src/templates/layouts/storyListing/index.test.tsx

```diff
@@ -1,5 +1,5 @@
 import { render, screen } from '@testing-library/react'
-import { StoryListing } from '@/templates/layouts/StoryListing'
+import { StoryListing } from '@/templates/layouts/storyListing'
```

Comment:
Noting this.

Reply:
yeah this not being camel case bugged me